### PR TITLE
Add DNS Forwarding as an Alternate Resolving Service to Connect Astro to AWS Data Sources

### LINKED ISSUE #1840: Document alternative setup for Route53
See https://astronomer.slack.com/archives/C015V2JFKT5/p1675378364584989

### PR DESCRIPTION
**Pull request by kwcanuck**
Resolves #1840

### CODE REVIEW

**Comment by kwcanuck** [comment]
@afastronomer thanks for getting this started! When time permits, can you review the new content and provide your feedback? Thanks!

File: astro/connect-aws.md [hunk 1]
```diff
@@ -218,3 +218,13 @@ To verify that the Amazon Route 53 Resolver rule was shared correctly, submit a
 #### Create a connection to confirm connectivity (optional)
 
 When Astronomer support confirms that the Amazon Route 53 Resolver rule was successfully associated with the Astro VPC, you can create a connection to the resource that is resolved by the shared rule. See [Managing Connections](https://airflow.apache.org/docs/apache-airflow/stable/howto/connection.html).
+
+### DNS Domain Name System forwarding
```
Comment (chomipi88) [inline astro/connect-aws.md:222]: I believe this is an alternative approach to using Route 53. Resolver rule would be option 1 and DNS forwarding is option 2.

Is that correct @afastronomer?

I believe this is the AWS doc for this second option: https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/resolver-forwarding-outbound-queries.html

File: astro/connect-aws.md [hunk 2]
```diff
@@ -218,3 +218,13 @@ To verify that the Amazon Route 53 Resolver rule was shared correctly, submit a
...
+
+You can use Domain Name System (DNS) forwarding as an alternate to using Route 53 Resolver rules to allow Astro to resolve DNS queries for resources running in other VPCs or on-premises. With DNS forwarding, you don't need to share your sensitive data with an external cloud account.
+
+To use this solution, make sure Astro is installed on the same  VPC peering or transit gateway subnets as the DNS server and then submit a request to [Astronomer support](https://cloud.astronomer.io/support). With your request, include the following information:
```
Comment (chomipi88) [inline astro/connect-aws.md:226]: Astro would not be on the same subnets as the DNS server. Instead, we need to ensure that Astro can reach/connect to the DNS server either through a VPC peer or TGW set up.

Correct @afastronomer ?

File: astro/connect-aws.md (hunk 1 above)
Comment (afastronomer) [reply astro/connect-aws.md:222]: Yes, it's an alternative but not a replacement for existing approach.

File: astro/connect-aws.md (hunk 2 above)
Comment (afastronomer) [reply astro/connect-aws.md:226]: Yes, DNS server locates on customer's side. Sometimes network\security team restricts access to it.

**Comment by chomipi88** [comment]
I'm not seeing the latest updates in the generated deploy preview

**Comment by kwcanuck** [comment]
@chomipi88 the latest revisions are now available. Refresh your browser to see the changes.

**Review by afastronomer — APPROVED** [review]

**Review by afastronomer — APPROVED** [review]

**Comment by chomipi88** [comment]
LGTM with the tab rename after discussion with Artem

**Comment by kwcanuck** [comment]
@jwitz ready for your review.

**Review by jwitz — APPROVED** [review]

File: astro/connect-aws.md [hunk 3]
```diff
@@ -180,20 +182,27 @@ Astronomer recommends using an external secrets backend to store your AWS access
 
 </Tabs>
 
-## Resolving services
+## Hostname resolution options
```
[the remark this answers is not in the document]
Comment (kwcanuck) [reply astro/connect-aws.md:185]: Done.

**Comment by chomipi88** [comment]
LGTM

**Pull request merged**